### PR TITLE
Update sv and svg info fetch to use V1 API

### PR DESCRIPTION
**Pull request by shifucun**
This is to use the new V1 REST API, which has better support (ex, custom DC sv node path).

Note here the routes are changed to match that of the REST API, with prefixes "/variable/" and "/variable-group/".

### CODE REVIEW

File: server/routes/api/variable.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import services.datacommons as dc
+
+bp = Blueprint("variable", __name__, url_prefix='/api/variable')
+
+
+@bp.route('/path')
+def get_variable_path():
+    """Gets the path of a stat var to the root of the stat var hierarchy.
+    """
+    dcid = request.args.get("dcid")
```
Comment (beets) [inline server/routes/api/variable.py:27]: nit: call html.escape on the dcid before further processing? it should not affect valid dcid's

**Review by beets — COMMENTED** [review]
nit: please add more detail to the pr description (especially with respect to the updates in routes).

File: server/config/topic_page_pb2.py [hunk 2]
```diff
@@ -2,10 +2,9 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
```
[the remark this answers is not in the document]
Comment (shifucun) [reply server/config/topic_page_pb2.py:2]: This is from running `protoc` with a newer version (3.21.9). I verified locally the topic page works fine.

File: server/routes/api/variable.py (hunk 1 above)
Comment (shifucun) [reply server/routes/api/variable.py:27]: This is directly passed to another url (instead of using as html text), so this should be fine?

**Comment by shifucun** [comment]
> nit: please add more detail to the pr description (especially with respect to the updates in routes).

Added more background

File: server/routes/api/variable.py (hunk 1 above)
Comment (chejennifer) [reply server/routes/api/variable.py:27]: I think the problem is that the dcid is directly returned as part of the response

File: server/routes/api/variable_group.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    """
+    dcid = request.args.get("dcid")
+    entities = request.args.getlist("entities")
+    url = "/v1/info/variable-group/{}".format(dcid)
+    if entities:
+        url += "?constrained_entities={}".format(entities[0])
+    if len(entities) > 1:
+        parts = list(
+            map(lambda e: "&constrained_entities={}".format(e), entities[1:]))
+        url += "".join(parts)
```
Comment (chejennifer) [inline server/routes/api/variable_group.py:45]: nit: could replace lines 40 - 45 with: 
```
if entities:
    url += "?constrained_entities=" + "&constrained_entities=".join(entities)
```

File: server/tests/api/api_stats_test.py [hunk 4]
```diff
@@ -236,12 +236,10 @@ def side_effect(query, places, sv_only):
             assert result == expected_sv_only_result
 
 
-class TestGetStatVarGroup(unittest.TestCase):
```
Comment (chejennifer) [inline server/tests/api/api_stats_test.py:239]: to keep things consistent, maybe we can extract these tests out into an "api_variable_test.py" and "api_variable_group_test.py" files?

File: server/routes/api/variable.py (hunk 1 above)
Comment (beets) [reply server/routes/api/variable.py:27]: yep that's the issue. and it gets `eval`ed by the browser so is a risk.

Comment (shifucun) [reply server/routes/api/variable.py]: Updated now

File: server/routes/api/variable_group.py (hunk 3 above)
Comment (shifucun) [reply server/routes/api/variable_group.py]: this is neat!

File: server/tests/api/api_stats_test.py (hunk 4 above)
Comment (shifucun) [reply server/tests/api/api_stats_test.py:239]: done

**Review by beets — APPROVED** [review]
thanks for the updates!

**Pull request merged**